API/src/microservices/Assessment-Service/index.js

```diff
@@ -1,8 +1,10 @@
 const { Assessments } = require(`../Database`);
 
-exports.submit = async (assessment) => {
-  // use the bookshelf model Assessments from API/src/microservices/Database to save
-  // the assessment data in the PostgreSQL database
+exports.submit = async (formatted_assessment) => {
+
+  Assessments.forge(formatted_assessment).save().then((a) => {
```

Comment:
```suggestion
  const new_assessment = Assessments.forge(formatted_assessment).save().then((a) => {
```
If you do this you can get the entire entry that it entered in the database. A key use of this is that you can get the id. With this you can pass it back to the frontend through the responses and function calls. If you end up having an assessment view page you can use the id to redirect from the form to the newly created assessment. I believe this is how bonfire does this.